cl/corpus_importer/utils.py

```diff
@@ -1024,3 +1025,44 @@ def get_court_id(raw_court: str) -> list[str]:
             return found_court
 
     return []
+
+
+def make_iquery_probing_key(court_id: str) -> str:
+    """Small wrapper to centralize iquery probing key semaphore generation.
+    :param court_id: A CL court ID
+    :return: The semaphore iquery probing key
+    """
+    return f"iquery.probing.enqueued:{court_id}"
+
+
+def compute_next_binary_probe(
+    iquery_pacer_case_id_final: int,
```

Comment:
Thanks for this suggestion. Yeah, I think it's better: `highest_known_pacer_case_id`. The final key is `iquery:highest_known_pacer_case_id` because I added the `iquery:` prefix to all the related Redis keys in this for better organization.